cmake/FindSparkle.cmake

```diff
@@ -0,0 +1,33 @@
+# -*- cmake -*-
+
+if (APPLE)
+  include(FindPackageHandleStandardArgs)
+
+  find_library(SPARKLE_FRAMEWORK
+    NAMES Sparkle
+    HINTS /usr/local/Caskroom/sparkle/1.22.0
```

Comment:
@rkitover I used `brew cask install sparkle` and it installed at this location. What can you tell me about this? Is this standard or some type of hack?

Reply:
Homebrew does things in their own directory and then symlinks it to /usr/local

Reply:
@ZachBacon Thank you for the info. We have moved from this solution, but it is good to know for future reference.